dotnet-authserver/src/TeacherIdentity.AuthServer/Models/Mappings/SmsConfirmationPinMapping.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace TeacherIdentity.AuthServer.Models.Mappings;
+
+public class SmsConfirmationPinMapping : IEntityTypeConfiguration<SmsConfirmationPin>
+{
+    public void Configure(EntityTypeBuilder<SmsConfirmationPin> builder)
+    {
+        builder.ToTable("sms_confirmation_pins");
+        builder.HasKey(p => p.SmsConfirmationPinId);
+        builder.Property(p => p.MobileNumber).HasMaxLength(100).IsRequired();
```

Comment:
100 characters should be enough. Most of the validation happens in notification service, so I don't want to be too strict